design-tokens/Package.swift

```diff
@@ -0,0 +1,17 @@
+import PackageDescription
+
+let package = Package(
+	name: "CompoundDesignTokens",
+	products: [
```

Comment:
We could expand the supported platforms in the future, but probably sensible for now to limit this to iOS (with the version matching ElementX's deployment target).
```suggestion
    platforms: [.iOS(.v16)],
	products: [
```